Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -876,6 +879,10 @@ static bool isElementExcludedByRule(const MarkupExclusionRule& rule, const Eleme
 
 bool MarkupAccumulator::shouldExcludeElement(const Element& element)
 {
+    if (element.document().requiresScriptExecutionTelemetry(ScriptTelemetryCategory::FormControls)) {
```

Comment:
Safer C++ is complaining that we're calling this non-trivial function (requiresScriptExecutionTelemetry) without storing document in a Ref/RefPtr.
I don't think we can call this function more than once per execution.
We should cache the result of this function call somewhere so that we don't keep repeatedly calling it.